src/panels/lovelace/hui-root.ts

```diff
@@ -150,9 +150,11 @@ class HUIRoot extends LitElement {
                     .narrow=${this.narrow}
                   ></ha-menu-button>
                   <div main-title>${this.config.title || "Home Assistant"}</div>
-                  <ha-start-voice-button
-                    .hass="${this.hass}"
-                  ></ha-start-voice-button>
+                  <paper-icon-button
```

Comment:
We should make sure it's only shown when the conversation integration is loaded.